### PR TITLE
Follow /unfollow User

### PR DESCRIPTION
**Pull request by slpriya**
Follow /unfollow User
LIke / Unlike Post
Get all details by Username For private and public route

### CODE REVIEW

**Review by Emily484 — APPROVED** [review]

**Review by solongoch — APPROVED** [review]
Looks good to merge

File: routes/api/follows.js [hunk 1]
```diff
@@ -3,62 +3,111 @@ const router = express.Router();
...
 
-
 // @route   put api/follow/:userId
 // @desc    Follow User
 // @input   Get Userid from req params of person whom user wants to follow.
 // @access  Private
 
 
-router.put('/follow/:userId' ,passport.authenticate('jwt', {session: false} ), (req, res) =>{
+router.put('/follow/:user_id', passport.authenticate('jwt', { session: false }), (req, res) => {
```
Comment (solongoch) [inline routes/api/follows.js]:
```suggestion
router.put('/:user_id/follow', passport.authenticate('jwt', { session: false }), (req, res) => {
```

File: routes/api/follows.js [hunk 2]
```diff
@@ -3,62 +3,111 @@ const router = express.Router();
...
 
 
-module.exports =router;
+// @route   put api/follow/:userId
+// @desc    Unfollow User
+// @input   Get Userid from req params of person whom user wants to unfollow.
+// @access  Private
 
 
+router.put('/unfollow/:user_id', passport.authenticate('jwt', { session: false }), (req, res) => {
```
Comment (solongoch) [inline routes/api/follows.js]:
```suggestion
router.put('/:user_id/unfollow', passport.authenticate('jwt', { session: false }), (req, res) => {
```

**Review by solongoch — APPROVED** [review]

**Comment by slpriya** [comment]
Merged Successfully 
Follow /unfollow User
Like / Unlike Post
Get all details by Username for both private and public route

**Pull request merged**